### PR TITLE
[Bots][Addedrequest headersofbot][1640875]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added new file request headers of the bot as per requirement 1640875 and 1357359 and added images and updated TOC. Please review.

### CODE REVIEW

File: msteams-platform/bots/how-to/conversations/request-headers-of-the-bot.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Request header fields
+
+Two non-standard request header fields are added to all the requests sent to bots, for both asynchronous flow and synchronous flow. The following table provides the request header fields and their values.
+
+| Field key | Value |
+|----------------|-----------------|
+| x-ms-conversation-id | The conversation ID corresponding to the request activity if applicable and confirmed or verified. |
+| x-ms-tenant-id | The tenant ID corresponding to the conversation in the request activity. |
+
+If the tenant or conversation ID is not present in the activity, the value is empty.
```
Comment (tacri) [inline msteams-platform/bots/how-to/conversations/request-headers-of-the-bot.md:20]: could we please add here " in the activity or was not validated on the service side, the value is empty"

File: msteams-platform/bots/how-to/conversations/request-headers-of-the-bot.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Send tenant ID and conversation ID to the request headers of the bot
+description: describes how to send tenant ID and conversation ID to the request headers of the bot.
+ms.topic: conceptual
+---
+
+# Send tenant ID and conversation ID to the request headers of the bot
+
+The current outgoing requests to the bot do not contain in the header or URL any information that helps bots route the traffic without unpacking the entire payload. The activities are sent to the bot to a URL similar to https://va.ai.dynamics.com/api/v1/messages/bots/1f0fe267-c3e3-4011-85fa-xxxxxxxxxx. Requests are received to show the conversation ID and tenant ID in the headers.
```
Comment (tacri) [inline msteams-platform/bots/how-to/conversations/request-headers-of-the-bot.md:9]: can we change please the URL to be more generic "https://<your_domain>/api/messages"?

**Review by tacri — DISMISSED** [review]

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**